frontend/src/metabase/app.js

```diff
@@ -3,7 +3,7 @@ import "regenerator-runtime/runtime";
 // This is conditionally aliased in the webpack config.
 // If EE isn't enabled, it loads an empty file.
 // Should be imported before any other metabase import
-import "ee-overrides"; // eslint-disable-line import/no-duplicates
+import "ee-overrides";
```

Comment:
Not sure why this passed our CI, but it failed locally.
![Screenshot 2025-02-18 at 2 10 54 PM](https://github.com/user-attachments/assets/abf2470d-b4f3-42ee-b73c-cf78205bc6eb)